api/agentpb/collector.proto

```diff
@@ -35,6 +35,8 @@ message MetricsBucket {
     string explain_fingerprint = 25;
     // ammount of variables in query.
     uint32 placeholders_count = 26;
+    // Contains extracted comments from queries.
+    repeated string comments = 27;
```

Comment:
why don't we make it a map? in that case whole logic of comments parsing would be in agent instead of splitting it across pmm-managed and pmm-agent

Reply:
Yes, right. Let me fix it.